Source/Renderer/AutomaticUniforms.js

```diff
@@ -1088,6 +1088,62 @@ define([
         }),
 
         /**
+         * Defines the top clipping plane.
+         *
+         * @alias czm_frustumTop
+         * @glslUniform
+         */
+        czm_frustumTop : new AutomaticUniform({
```

Comment:
Put these four frustum planes into a single `vec4` like `czm_viewport`.  It will have better performance.